integration-test/trace-recorder/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+repositories {
+    mavenCentral()
+    maven { url = uri("https://repo.gradle.org/gradle/libs-releases/") }
+}
+
+sourceSets {
+    main {
+        java.srcDirs("src/main")
+
+        resources {
+            srcDir("src/main/resources")
+        }
+    }
+
+    dependencies {
+        val slf4jVersion: String by project
+
+        implementation(project(":integration-test:common"))
+        implementation("org.slf4j:slf4j-simple:$slf4jVersion")
+    }
+}
+
+tasks {
+    processResources {
+        duplicatesStrategy = DuplicatesStrategy.INCLUDE
+    }
+
+    registerTraceAgentIntegrationTestsPrerequisites()
+
+    val traceRecorderIntegrationTest = register<Test>("traceRecorderIntegrationTest") {
+        configureJvmTestCommon(project)
+        group = "verification"
+        include("org/jetbrains/test/trace/recorder/*")
+
+        testClassesDirs = sourceSets["main"].output.classesDirs
+        classpath = sourceSets["main"].runtimeClasspath
```

Comment:
I think previously it was for `rootProject`'s `main` source set. Do we still need it?

Reply:
So ':test' task runs tests from all subprojects, right?
Is there any simpler way to disable this behavior, apart from moving tests to `main` source set?